docs/config.rst

```diff
@@ -1086,8 +1086,22 @@ user can bypass the system's interpreter parser by defining the
 When the workaround is enabled, all tox-invoked text file executables will have
 their interpreter directive parsed by and explicitly executed by ``tox``.
 
+Environment variables
+---------------------
+tox will treat the following environment variables:
+- ``TOX_DISCOVER`` for python discovery first try the python executables under these paths
+- ``TOXENV`` see envlist.
+- ``TOX_LIMITED_SHEBANG`` see Advanced settings > Handle interpreter directives with long lengths.
+- ``TOX_PARALLEL_NO_SPINNER`` see Parallel mode.
+- ``_TOX_PARALLEL_ENV`` lets tox know that it is invoked in the parallel mode.
+- ``TOX_PROVISION`` is only intended to be used internally.
+- ``TOX_REPORTER_TIMESTAMP`` enables showing for each output line its delta
+since the tox startup when set to ``1``.
```

Comment:
I do not understand this sentence :-/

Reply:
Do you mean the sentence for `TOX_REPORTER_TIMESTAMP`?

Reply:
Yes :-/